cpp/thirdparty/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 # If you like to upgrade, just change the number
 set(MEMILIO_EIGEN_VERSION "3.3.9")
 set(MEMILIO_SPDLOG_VERSION "1.11.0")
+set(MEMILIO_BOOST_VERSION "1.84.0") # Minimal version needed: 1.76.0 

```

Comment:
1.76.0 intended as a comment?

Reply:
the comment seems very likely to go stale though.

Reply:
added this to the README and included it as a variable for the find package command